Tests/CADCacheAdvanceTests/CADCacheAdvanceTests.m

```diff
@@ -0,0 +1,96 @@
+//
+//  Created by Dan Federman on 4/24/20.
+//  Copyright © 2020 Dan Federman.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS"BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+#if GENERATED_XCODE_PROJECT
+// These imports work when working within the generated CacheAdvance.xcodeproj used in CI.
+#import <CADCacheAdvance/CADCacheAdvance-Swift.h>
+#import <XCTest/XCTest.h>
```

Comment:
I kind of wonder if we could use `@import XCTest` in both cases.
The `-Swift.h` makes more sense as something that needs to be different.

Reply:
I believe we can. I thought there was some benefit to symmetry but maybe not. Updated.

Reply:
We're definitely splitting hairs at this point haha